app/views/openid_connect/logout/index.html.erb

```diff
@@ -2,7 +2,11 @@
 
 <div class='text-center'>
   <%= image_tag(asset_url('user-access.svg'), width: '280', height: '91', alt: '') %>
-  <%= render PageHeadingComponent.new.with_content(t('openid_connect.logout.heading', app_name: APP_NAME)) %>
+  <% if @service_provider %>
```

Comment:
This is kind of a double check for whether the service provider exists and has a friendly name.  I'm not sure if all SPs have a friendly name, but there's a future where we support OIDC Logout without redirecting and this block will be useful then too.

@orenyk do all SPs have a friendly name?